content/en/docs/collector/builder.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Builder
+weight: 25
+---
+
+The OpenTelemetry Collector Builder (ocb) is a command-line interface tool that
+generates sources and binaries for OpenTelemetry Collector
+[distributions](../distributions).
```

Comment:
Putting the purpose of the tool in the opening paragraph will help users understand when they should use the builder sooner.

```suggestion
The OpenTelemetry Collector Builder (ocb) is a command-line interface tool that
allows users to build a custom [distribution](../distributions) for the OpenTelemetry
Collector.
```